md/single-sign-on-with-saml.md

```diff
@@ -0,0 +1,214 @@
+# Single sign-on with SAML
+
+:::info 
+**Note:** For Performance, Efficiency, and Teamwork editions only.
+:::
+*
+This pages explains how to configure your Bonita BPM Platform system to use SAML to provide single sign-on (SSO). It assumes you already have a working SAML service. All Bonita BPM users must be registered in SAML.
+
+This information applies to a Bonita BPM platform deployed from a bundle, not to the Engine launched from Bonita BPM Studio.
+
+SAML configuration is at tenant level. Each tenant can use a different SAML service.
+
+## SAML overview for Bonita BPM 
+
+This is an overview that relates the steps required to integrate a bonita BPM bundle with an SAML identity provider.
+
+![](images/saml-overview.png)<!--{.img-responsive}-->
```

Comment:
I think it's a good practice to give a title to the images (Authentication over SAML)